dist/infragram.js

```diff
@@ -587,14 +548,9 @@ function _slicedToArray(arr, i) { if (Array.isArray(arr)) { return arr; } else i
         lnk.download = new Date().toISOString().replace(/:/g, "_") + "." + format; // create a "fake" click-event to trigger the download
 
         if (document.createEvent) {
-          var mouseEvent = _getMouseEvent();
-
-          lnk.dispatchEvent(mouseEvent("click", {
-            screenX: 0,
-            screenY: 0,
-            clientX: 0,
-            clientY: 0
-          }));
+          event = document.createEvent("MouseEvents");
```

Comment:
I believe the above changes were made in https://github.com/publiclab/infragram/pull/324 so we should leave it as it is with dispatchEvent. Thank you!